README.md

```diff
@@ -11,38 +12,101 @@ pip install hatebase
 pip install requests
 ```
 
+# HatebaseAPI functions
+### Initialize HatebaseAPI class
+```
+hatebase = HatebaseAPI({"key": key})
+```
+
+### HatebaseAPI getVocabulary
+```
+# set filters for vocabulary query
+filters = {"language": "eng"}
+output = "json"
+
+# get response tuple from query
+# response[0] = json_response
+# response[1] = vocabulary_list of that page
+# response[2] = total nr of pages 
+# response[3] = total nr of vocabulary entries
+# response[4] = language
+response = hatebase.getVocabulary(filters=filters, format=output)
+
+## alternatively, get the response details directly
+json_response, vocab, pages, total_entries, lang = hatebase.getVocabulary(filters=filters, format=output)
```

Comment:
I think I probably made a mistake in the previous version by requiring a user to manually convert the response from a JSON string into a Python object.  It seems as if the answers are only returned in JSON format.  @meierale , do you think it would make sense to just remove the format option and have all responses automatically return the result of `response.json()`?  Open to conversation here :-)

Reply:
Hi @DanielJDufour. The Hatebase API documentation states "At this time only JSON format is available." So for the current version we could indeed omit the format parameter, or set it do json as default value. As currently it is all json anyways, we could indeed just return the `response.json()` all the time.

But it also seems like hatebase intends to offer different formats in the future. Once that is the case, we would have to adjust the code again.